rc/lsp.kak

```diff
@@ -1415,6 +1417,26 @@ incomingOrOutgoing = $1
 " | eval "${kak_opt_lsp_cmd} --request") > /dev/null 2>&1 < /dev/null & }
 }
 
+define-command -hidden lsp-breadcrumbs-request -docstring "request updating lsp_modeline_breadcrumbs for the buffer" %{
```

Comment:
I've addressed all issues I found important enough and pushed over it (previous version was 9b36bb56ae1526fdaf0de922c7953cf398c94825).
If you have feedback, let me know.

Otherwise I'll merge later today.
I'm really happy about this feature, I've wanted it for years.
I think it makes a big difference in my workflow.
Sure do do have commands to jump to symbol start but I don't want to clobber my selectin/jump history just for finding the function name.

Reply:
@krobelus Oh, sorry, I think I've force pushed as I didn't notice your messages. Can you force push over again?